### DISCUSSION TITLE
how to contribute

### DISCUSSION THREAD

**Discussion by ahammadunny** [sButtons/sbuttons#1219] [Q&A] [answered]
Hi,
I  am somewhat new to GITHUB. I want to contribute to SBUTTONS, as a learning experience. I know how to create a local repository on my machine. But I do not know how to work with the imported repository. For example, I can't see the buttons displayed. where should I start to pick ip a button so that I can add animated class.? Hope to hear from you.

Ahammad unny Pathiaseril

**Comment by shahednasser** [comment, marked as answer]
hello @ahammadunny ,

to contribute to this repository, you need first to fork it then clone it on your local machine. Then, please check out the issues section for issues to work on or the ideas category here in discussions. I suggest you go through the [contributing guidelines](https://github.com/sButtons/sbuttons/blob/7e54e1d080e330fed502779b29c022a704a78071/CONTRIBUTING.md) to understand how you can contribute to different parts of the repository, then ask me if you have any questions.

Thank you!
